autoware_adapi_v1_msgs/vehicle/msg/VehicleKinematics.msg

```diff
@@ -0,0 +1,9 @@
+# Geographic point, using the WGS 84 reference ellipsoid.
+# Currently only support conversion from single MGRS Grid code or UTM
+# If the vector map do not contain correct MGRS Grid code or UTM, this data will be not valid
```

Comment:
The comments in message file should be focusing on the description of what kind of information would be available in the message.
I don't think supported conversion type should be included in the message file, but it should be in the README of the node that is doing the conversion. (I don't want people to modify this comment whenever they add new projection type to Autoware)
```suggestion
# Geographic point, using the WGS 84 reference ellipsoid.
# This data will be invalid If Autoware does not provide projection information between geographic coordinates and local coordinates. 
```